**Hvorfor skjer det fusjon på innsiden av stjerner?**

Vi kan finne andelen av partiklene som har den nødvendige farten ved å integrere Maxwell-Boltzmannfordelingen, fra kritisk hastighet til uendelig.

Den kritiske hastigheten til partiklene $v_{rms}$ ved $2.3 \times 10^9 K$ finner vi ved formelen $v_{rms} = \sqrt{\frac{3k_B T}{m}}$

Andelen $P$ som er over den kritiske hastigheten $v_{rms}$ er gitt ved integalet av sannsynlighetsfordelingen, 
ved temperatur $T = 1.5 \times 10^7 K$

$$P = \frac{4\pi}{\left( \frac{2\pi k_B T}{m} \right)^{3/2}} \int_{v_{rms}}^\infty v^2 e^{-\frac{mv^2}{2k_B T}} dv$$

In [1]:
import numpy as np
kB = 1.38064852e-23
T = 1.5e7
m = 1.67e-27
v_rms = lambda T: np.sqrt(3*kB*T/m)
kritisk_hastighet = v_rms(2.3e9)

def MB_speed(v):
    return (m/(2*np.pi*kB*T))**(3/2)*4*np.pi*(v**2)*np.exp(-m*v**2/(2*kB*T))

integrasjons_område = np.linspace(kritisk_hastighet,100*kritisk_hastighet,10000)

print("Andel som har større enn kritisk hastighet: ", np.trapz(MB_speed(integrasjons_område)))

Andel som har større enn kritisk hastighet:  6.891535245365216e-104


Ettersom det kun er $9.21 \times 10^{53}$ atomer i solen (`hentet fra google`), vil vi ikke forvente at det er noen partikler med en hastighet større en den kritiske farten, ettersom andelen er av størrelsesorden $10^{-104}$

Det er derfor sannsynlig at det er et annet mekanisme som er ansvarlig for fusjonen i stjerner. Mest sannsynlig vil dette være trykket som partiklene utøver på hverandre i kjernen.